src/Response/Initializer/Initializer.php

```diff
@@ -94,6 +94,17 @@ function () use ($profiler) {
                     );
                 }
 
+                //add plugins from config
+                if (isset($dep['Maleficarum\Config']['response']['plugins']) && is_array($dep['Maleficarum\Config']['response']['plugins'])) {
+                    foreach ($dep['Maleficarum\Config']['response']['plugins'] as $plugin) {
+                        $p = \Maleficarum\Ioc\Container::get($plugin);
+                        if (!$p instanceof \Maleficarum\Response\Plugin\AbstractPlugin) {
+                            throw new \LogicException('Invalid plugin type specified.');
+                        }
+                        $responseHandler->addPlugin($p->getName(), $p->getClosure());
```

Comment:
I don't quite get why we have a nice "Plugin" class and add just it's closure when we want to use it?